docs/building-on-tigerbeetle/data-modeling.md

```diff
@@ -4,23 +4,35 @@ sidebar_position: 1
 
 # Data Modeling
 
-TigerBeetle is a domain-specific database — its schema of [`Account`s](../reference/accounts.md) and
-[`Transfer`s](../reference/transfers.md) is built-in and fixed. In return for this prescriptive
-design, it provides excellent performance, integrated business logic, and powerful invariants.
+This section describes various aspects of the TigerBeetle data model and provides some suggestions
+for how you can map your application's requirements onto the data model.
 
-This section is a sample of techniques for mapping your application's requirements onto
-TigerBeetle's data model. Which (if any) of these techniques are suitable is highly
-application-specific.
+## Accounts, Transfers, and Ledgers
 
-When possible, round trips and coordination can be minimized by encoding application invariants
-directly in TigerBeetle rather than implementing them in the application itself (or with a foreign
-database). This is useful for both maintaining consistency and performance.
+The TigerBeetle data model consists of [`Account`s](../reference/accounts.md),
+[`Transfer`s](../reference/transfers.md), and ledgers.
+
+### Ledgers
+
+Ledgers partition accounts into groups that may represent a currency or asset type or any other
+logical grouping. Only accounts on the same ledger can transact directly, but you can use atomically
+linked transfers to implement [currency exchange](./recipes/currency-exchange.md).
```

Comment:
Link "linked" and [linked](https://docs.tigerbeetle.com/design/client-requests#linked-events)?

```suggestion
[linked](./client-requests.md#linked-events) transfers to implement [currency exchange](./recipes/currency-exchange.md).
```